packages/nuxi/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Special exit code to restart the process
+ *
+ * Usage:
+ * ```ts
+ * if (process.env.NUXI_CLI_WRAPPER) {
+ *   process.exit(EXIT_CODE_RESTART)
+ * }
+ * ```
+ */
+export const EXIT_CODE_RESTART = 44
```

Comment:
Although I assume this implementation is temporary, code 44 is reserved for UNIX (Channel number out of range). As we don't (and shouldn't) use code 0, i think we can simply use that code as an indicator that child was killed and expects to be restarted (otherwise use a higher number)

Reply:
I was checking this table: https://tldp.org/LDP/abs/html/exitcodes.html says 3-125 are free to use.

And yes, 44 was a number I picked randomly.

Go a bit deeper I found this:
https://www.cyberciti.biz/faq/linux-bash-exit-status-set-exit-statusin-bash/
It seems to me the codes are specially for bash to exit, I suppose it shouldn't have much impact on our side (+ the code is only be captured by our wrapper)

So anyway I changed it to 85 as it's defined in the previous link as `Interrupted system call should be restarted`

Reply:
We could use 1. Auto restarting with child unwanted exit codes too as well as signaling explicitly (also consistent with sighup used for restart)

Reply:
I am not sure if I get what you mean - are you suggesting we exit with `process.exit(1)`? Wouldn't that be indistinguishable with the normal error exit?

Reply:
A child normally doesn't exits with (1) but yeah it might introduce edge-cases.

Well, btw if we use fork, we can use the process communication channel to explicitly send a nuxt event (https://github.com/nuxt/cli/pull/22/files#diff-34d76a96faa930c4b9909c055beb422e601e2267f72ce12d6f15a1c2cc6ea914R57)